resources/ssdlc/compliance_report_template.md

```diff
@@ -0,0 +1,30 @@
+
+# Atlas SQL schema manager SSDLC Compliance Report - %VERSION%
+
+**Release date**
+%RELEASE_DATE%
+
+**Release Creator**  
+%AUTHOR% - %AUTHOR_EMAIL%
+
+**Process Document**  
+https://www.mongodb.com/resources/products/capabilities/supply-chain-security-in-mongodb-s-software-development-lifecycle
+
+**Tool used to track third party vulnerabilities**  
+https://osv.dev/
+
+**Third-Party Dependency Information**  
+See SBOM at URL: %SBOM_URL%
+
+**Static Analysis Findings**  
+See report at URL: %SARIF_URL%
+
+**Signature Information**
+Product is signed with signatures available upon request. 
+To verify a package, request the signature artifacts matching your library and follow the instructions to validate MongoDB packages described [here](https://www.mongodb.com/docs/manual/tutorial/verify-mongodb-packages/). 
+
+> :warning:  
+> For this [step](!https://www.mongodb.com/docs/manual/tutorial/verify-mongodb-packages/#download-then-import-the-key-file), download and import the MongoDB schema manager public key using this url : `https://pgp.mongodb.com/mongodb-schema-manager.asc`
```

Comment:
I think this is for images?

```suggestion
> For this [step](https://www.mongodb.com/docs/manual/tutorial/verify-mongodb-packages/#download-then-import-the-key-file), download and import the MongoDB schema manager public key using this url : `https://pgp.mongodb.com/mongodb-schema-manager.asc`
```

Reply:
Yep 
Oliver noted it on the other PR, the change is already in the oven, I just didn't have time to push it, you were too quick to review :)